pkg/agent/flowexporter/connections/conntrack_windows.go

```diff
@@ -17,11 +17,45 @@
 package connections
 
 import (
+	"fmt"
 	"net"
+	"strconv"
+	"strings"
 
 	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
 )
 
-func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDRv4 *net.IPNet, serviceCIDRv6 *net.IPNet, isAntreaProxyEnabled bool) *connTrackOvsCtl {
-	return NewConnTrackOvsAppCtl(nodeConfig, serviceCIDRv4, serviceCIDRv6, isAntreaProxyEnabled)
+type connTrackOvsCtlWindows struct {
+	connTrackOvsCtl
+}
+
+// dpctl/ct-get-maxconns returns operation not supported on Windows node, use dpctl/ct-get-limits intead.
```

Comment:
That's not a good function-level comment. It belongs with the call to `ct.ovsctlClient.RunAppctlCmd`

Reply:
Thanks, addressed.